client/js/components/procedure/basicSettings/AutoSwitchProcedurePhaseForm.vue

```diff
@@ -64,10 +64,10 @@
             enforce-plausible-dates
             :min-date="startDate"
             required
-            :data-cy="{
+            :data-cy="JSON.stringify({
                 endDate: dataCyEndDate,
                 startDate: dataCyStartDate
-            }"
+            })"
```

Comment:
I think it would be better to either directly change the type to an object in the DateRangePicker or to only pass a string, such as `data-cy="newProcedureForm"`. This is because in DpDateRangePicker at [lines 14 and 28]([url](https://github.com/demos-europe/demosplan-ui/blob/main/src/components/DpDateRangePicker/DpDateRangePicker.vue#L14)), we are already passing `${dataCy}:startDate` and `${dataCy}:endDate`.

Reply:
Thanks, changed in [38972aa](https://github.com/demos-europe/demosplan-core/pull/3086/commits/38972aa27cd8f569f68b315b090a6107fc08b0a9)